test/unit/registration.test.js

```diff
@@ -0,0 +1,123 @@
+import Microcosm from '../../src/microcosm'
+import { ALIASES } from '../../src/get-registration'
```

Comment:
thoughts on renaming ALIASES to STATUSES? or would that possibly be a breaking change